src/System.Net.WebHeaderCollection/ref/System.Net.WebHeaderCollection.cs

```diff
@@ -88,34 +88,34 @@ public partial class WebHeaderCollection : System.Collections.Specialized.NameVa
     {
         public WebHeaderCollection() { }
         protected WebHeaderCollection(System.Runtime.Serialization.SerializationInfo serializationInfo, System.Runtime.Serialization.StreamingContext streamingContext) { }
-        public override string[] AllKeys { get { throw null; } }
+        public override string?[] AllKeys { get { throw null; } }
         public override int Count { get { throw null; } }
-        public string this[System.Net.HttpRequestHeader header] { get { throw null; } set { } }
-        public string this[System.Net.HttpResponseHeader header] { get { throw null; } set { } }
-        public new string this[string name] { get { throw null; } set { } }
+        public string? this[System.Net.HttpRequestHeader header] { get { throw null; } set { } }
+        public string? this[System.Net.HttpResponseHeader header] { get { throw null; } set { } }
+        public new string? this[string name] { get { throw null; } set { } }
         public override System.Collections.Specialized.NameObjectCollectionBase.KeysCollection Keys { get { throw null; } }
-        public void Add(System.Net.HttpRequestHeader header, string value) { }
-        public void Add(System.Net.HttpResponseHeader header, string value) { }
+        public void Add(System.Net.HttpRequestHeader header, string? value) { }
+        public void Add(System.Net.HttpResponseHeader header, string? value) { }
         public void Add(string header) { }
-        public override void Add(string name, string value) { }
-        protected void AddWithoutValidate(string headerName, string headerValue) { }
+        public override void Add([System.Diagnostics.CodeAnalysis.DisallowNull] string? name, string? value) { }
```

Comment:
Nullable because of overwriting parent signature, but throw on null in the implementation, hope [DisallowNull] is correct attribute in this case

Reply:
This should just be `string name`.  You'll get a warning that you'll need to suppress.